src/components/side_panel/chart_panel.ts

```diff
@@ -239,7 +242,7 @@ export class ChartPanel extends Component<Props, SpreadsheetEnv> {
   private updateChart(definition: ChartUIDefinitionUpdate): DispatchResult {
     return this.env.dispatch("UPDATE_CHART", {
       id: this.props.figure.id,
-      sheetId: this.getters.getActiveSheetId(),
+      sheetId: this.state.chartSheetId,
```

Comment:
sheetId in this command is still unused, right ?

Reply:
not here ,but it's actually used starting v16.0